diff --git a/node_modules/@dfinity/agent/lib/esm/vendor/bls/bls.js b/node_modules/@dfinity/agent/lib/esm/vendor/bls/bls.js
index 0726cde..08867ba 100644
--- a/node_modules/@dfinity/agent/lib/esm/vendor/bls/bls.js
+++ b/node_modules/@dfinity/agent/lib/esm/vendor/bls/bls.js
@@ -1,5 +1,6 @@
 /* tslint:disable */
 /* eslint-disable */
+import { decode } from 'base64-arraybuffer'
 let wasm;
 // This WASM is generated from the BLS Rust code of the Agent RS (see
 // http://github.com/dfinity/agent-rs/)
@@ -802,7 +803,7 @@ const wasmBytesBase64 = `
     kLWJ5AwVydXN0Yx0xLjQ5LjAgKGUxODg0YThlMyAyMDIwLTEyLTI5KQZ3YWxydXMGMC4xOC4wDHdhc20tYmluZGdlbhIw
     LjIuNzAgKGI2MzU1YzI3MCk=
 `.replace(/[^0-9a-zA-Z/+]/g, '');
-const wasmBytes = require('base64-arraybuffer').decode(wasmBytesBase64);
+const wasmBytes = decode(wasmBytesBase64);
 /**
  * @returns {number}
  */
